### PR TITLE
Remove brackets from PHP statements

### PR DESCRIPTION
**Pull request by grandeljay**
This PR changes some mentions of PHP statements which have brackets on them. It makes it looks like they are functions which they are clearly not. VS Code correctly describes them as statements. The tooltips should now reflect that too.

image

### CODE REVIEW

**Comment by roblourens** [comment]
Sure, and I'm not a PHP expert but it seems common and idiomatic to include parens, even if they aren't required, right? Plus it seems that PHP includes them in error messages. If a `require` fails, I get `Fatal error: require(): Failed opening required`

**Comment by grandeljay** [comment]
That is a very good point. I was browsing the official PHP doc and I found some arguments for my PR:

* [include](https://www.php.net/manual/en/function.include.php)
  > Because include is a special language construct, parentheses are not needed around its argument.
  
* [require](https://www.php.net/manual/en/function.require.php)
  > Remember, when using require that it is a statement, not a function.

  (found in the top voted user contribution)
  

As you may know PHP is a heavily criticised programming language and often rightfully criticized. There are many quirks and inconsistencies. I'm willing to bet the error message you received (`require()` instead of `require`) is also incorrect.

File: extensions/php-language-features/src/features/phpGlobals.ts [hunk 1]
```diff
@@ -198,13 +198,13 @@ export const keywords: IEntries = {
...
+		description: 'If called from within a function, the return statement immediately ends execution of the current function, and returns its argument as the value of the function call. return will also end the execution of an eval() statement or script file. \r\n\r\nIf called from the global scope, then execution of the current script file is ended. If the current script file was includeed or requireed, then control is passed back to the calling file. Furthermore, if the current script file was includeed, then the value given to return will be returned as the value of the include call. If return is called from within the main script file, then script execution ends. If the current script file was named by the auto_prepend_file or auto_append_file configuration options in php.ini, then that script file\'s execution is ended.',
```
Comment (roblourens) [inline extensions/php-language-features/src/features/phpGlobals.ts:207]: Please fix the misspellings then I can merge

**Comment by grandeljay** [comment]
I also edited `echo` now

**Pull request merged**